WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectedReaderView.swift

```diff
@@ -1,10 +1,8 @@
 import UIKit
+import Yosemite
 
 final class CardReaderSettingsConnectedReaderView: NSObject {
-
-    /// A simple model for this "ViewModel" - just a reference to the CardReaderSettingsViewModel connected reader
-    var connectedReader: CardReader?
-
+    var viewModel = CardReaderViewModel()
```

Comment:
Sorry. I missed this in previous passes. If the viewModel is not going to be modified, it's probably better if it's declared as `let` or even better `private let`

Reply:
Hmmmm. Good point! I want to change all the ViewModels to be passed in and will address this in #3744